context/package-lock.json

```diff
@@ -15694,6 +15694,11 @@
       "integrity": "sha512-mT8iDcrh03qDGRRmoA2hmBJnxpllMR+0/0qlzjqZES6NdiWDcZkCNAk4rPFZ9Q85r27unkiNNg8ZOiwZXBHwcA==",
       "dev": true
     },
+    "chart.js": {
+      "version": "3.4.1",
+      "resolved": "https://registry.npmjs.org/chart.js/-/chart.js-3.4.1.tgz",
+      "integrity": "sha512-0R4mL7WiBcYoazIhrzSYnWcOw6RmrRn7Q4nKZNsBQZCBrlkZKodQbfeojCCo8eETPRCs1ZNTsAcZhIfyhyP61g=="
+    },
```

Comment:
Just to keep things tidy, I'd say revert this now, and we can bring it back when you need it. It's good to keep PRs on topic, and not let unrelated changes creep in.
```suggestion
```
... or, alternatively, add the charting to this PR... but I would lean towards keeping it small.